devicetypes/aamann/enlighten-envoy-local.src/enlighten-envoy-local.groovy

```diff
@@ -296,9 +302,10 @@ def ping() {
 
 def startPoll() {
 	unschedule()
-	// Schedule 2 minute polling
+	def wantMin = 6       // Schedule N minute polling Integer < 60
```

Comment:
Here is an example log - even with 2 minute polling the health check gets triggered as the `parse()` function is not called reliably…
```
UUID  11:00:09 AM: debug Solar - requesting latest data from Envoy via JSON API…
UUID  10:58:15 AM: debug Solar - new data: [wattHoursToday:9024, wattsNow:3123, wattHoursLifetime:21590707, wattHoursSevenDays:156031]
UUID  10:58:15 AM: debug Solar - requesting latest data from Envoy via JSON API…
UUID  10:58:15 AM: trace Solar - checking device health…
UUID  10:58:09 AM: debug Solar - requesting latest data from Envoy via JSON API…
UUID  10:56:10 AM: debug Solar - requesting latest data from Envoy via JSON API…
UUID  10:54:09 AM: debug Solar - requesting latest data from Envoy via JSON API…
UUID  10:52:09 AM: debug Solar - requesting latest data from Envoy via JSON API…
UUID  10:50:09 AM: debug Solar - requesting latest data from Envoy via JSON API…
UUID  10:48:10 AM: debug Solar - no new data
UUID  10:48:09 AM: debug Solar - requesting latest data from Envoy via JSON API…
UUID  10:46:09 AM: debug Solar - requesting latest data from Envoy via JSON API…
UUID  10:44:10 AM: debug Solar - new data: [wattHoursToday:8237, wattsNow:3434, wattHoursLifetime:21589920, wattHoursSevenDays:156058]
UUID  10:44:09 AM: debug Solar - requesting latest data from Envoy via JSON API…
```